### PR TITLE
Add self-hosted server button to /servers

### PR DESCRIPTION
**Pull request by Novampr**
Covers over PaperMC, FabricMC and Forge

Tells you how to join both locally and globally

Tells you 2 different ways you may join globally, port-forwarding and playit

### CODE REVIEW

**Comment by DevLimeAtSkyo** [comment]
I gave up on the icon as you can see 💀

File: buttons.js [hunk 1]
```diff
@@ -24,6 +24,10 @@ const buttons = {
             .setLabel('Bedrock players')
             .setEmoji("<:bedrock:1089270506441687221>")
             .setStyle(ButtonStyle.Danger),
+        new ButtonBuilder()
+            .setCustomId('self_hosted')
```
Comment (TecEash1) [inline buttons.js:28]: Please add an icon here, you can use .setEmoji, if you want a custom one please let us know and we can send u an emoji id :)

File: embeds.js [hunk 2]
```diff
@@ -266,7 +266,45 @@ const embeds = {
     
     devs_only: new EmbedBuilder()
     .setDescription("Only developers of <@997670790604542012> can use this.")
-    .setColor("ED4245")
+    .setColor("ED4245"),
+    selfhostserver: new EmbedBuilder() //If you want to make more complex embeds like this I recommend https://embed.dan.onl/ a-m-a-z-i-n-g
```
Comment (TecEash1) [inline embeds.js:270]: try to shorten this please, like how the other command is written

**Review by TecEash1 — CHANGES_REQUESTED** [review]

File: buttons.js (hunk 1 above)
Comment (DevLimeAtSkyo) [reply buttons.js:28]: alr gimme a sec

File: embeds.js (hunk 2 above)
Comment (DevLimeAtSkyo) [reply embeds.js:270]: Is this commit short enough?

File: buttons.js [hunk 3]
```diff
@@ -27,6 +27,7 @@ const buttons = {
         new ButtonBuilder()
             .setCustomId('self_hosted')
             .setLabel('Self Hosted server')
+            .setEmoji("<:log:1096054676903956560>")
```
Comment (TecEash1) [inline buttons.js:30]: instead of adding it like that would you mind sending us the emoji so we can upload it and give u the id?

**Review by TecEash1 — CHANGES_REQUESTED** [review]

File: embeds.js (hunk 2 above)
Comment (TecEash1) [reply embeds.js:270]: ye looks good!

File: buttons.js (hunk 3 above)
Comment (TecEash1) [reply buttons.js:30]: it might, might not. Better to be safe than sorry. I can send u the id after anyway

Comment (TecEash1) [reply buttons.js:30]: o wait its a literal log lmao from the qc server

Comment (TecEash1) [reply buttons.js:30]: could you maybe get a custom emoji for a self hosted server guide?

Comment (TecEash1) [reply buttons.js]: https://github.com/Cartrigger/QCXRSoonBot/pull/51/commits/1f9d5b01a36fe9fc6693ee4a6bd2f9c66be75fa3 Fixed it :)

**Comment by DevLimeAtSkyo** [comment]
Alr does this look good Techy?

**Comment by DevLimeAtSkyo** [comment]
why does it say changes requested, they were resolved 😭

**Review by Novampr — COMMENTED** [review]
????
pr going funky

**Comment by TecEash1** [comment]
discord, sorry lime maybe like if people ask a bit more 

Excellent work tho looks great, ty anyway!

**Review by Novampr — COMMENTED** [review]
bye review, this has been bugging me

**Pull request closed without merging**